clippy_lints/src/append_instead_of_extend.rs

```diff
@@ -0,0 +1,61 @@
+use rustc_session::{declare_lint_pass, declare_tool_lint};
+// use rustc_ast::ast::*;
+use clippy_utils::diagnostics::span_lint_and_help;
+use clippy_utils::ty::is_type_diagnostic_item;
+use if_chain::if_chain;
+use rustc_hir::{Expr, ExprKind};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_span::symbol::sym;
+
+declare_clippy_lint! {
+    /// **What it does:**
+    ///
+    /// **Why is this bad?**
+    ///
+    /// **Known problems:** None.
+    ///
+    /// **Example:**
+    ///
+    /// ```rust
+    /// // example code where clippy issues a warning
+    /// ```
+    /// Use instead:
+    /// ```rust
+    /// // example code which does not raise clippy warning
+    /// ```
```

Comment:
You've now committed the default lint documentation and description in your latest commit. You can leave this for now, this is just a reminder, to make sure we don't forget :)